cloud/scope/cluster_test.go

```diff
@@ -254,12 +261,9 @@ func TestNewClusterScope(t *testing.T) {
 					LinodeCluster: &infrav1alpha1.LinodeCluster{},
 				},
 			},
-			expectedError: fmt.Errorf("failed to init patch helper: obj is nil"),
-			getFunc: func(ctx context.Context, key types.NamespacedName, obj *corev1.Secret, opts ...client.GetOption) error {
-				return nil
-			},
-			patchFunc: func(obj client.Object, crClient client.Client) (*patch.Helper, error) {
-				return nil, fmt.Errorf("obj is nil")
+			expectedError: fmt.Errorf("failed to init patch helper: no kind is registered for the type v1alpha1.LinodeCluster in scheme \"pkg/runtime/scheme.go:100\""),
```

Comment:
We may want to just check for a substring of the error string since `pkg/runtime/scheme.go:100` could change.